app/helpers/application_helper.rb

```diff
@@ -1,9 +1,13 @@
 module ApplicationHelper
-  def session_pill_class(session_date = nil)
+  def active_pill_class(session_date = nil)
     if request.params[:on_date] == session_date.to_s || (session_date.blank? && request.params[:on_date].blank?)
       "bg-red text-white"
     else
       "bg-white text-red"
     end
   end
+
+  def active_filters?
+    SessionQuery::STATUS_SCOPES.any? { |status| request.params[status] == "1" }
+  end
```

Comment:
Could we move this to a `SessionsHelper` module and keep the method names scoped to "sessions"? (given that they are global)